SeaPublicWebsite/Helpers/BasicAuthMiddleware.cs

```diff
@@ -3,16 +3,19 @@
 using System.Text;
 using System.Threading.Tasks;
 using Microsoft.AspNetCore.Http;
+using Microsoft.Extensions.Configuration;
 
 namespace SeaPublicWebsite.Helpers
 {
     public class BasicAuthMiddleware
     {
         private readonly RequestDelegate next;
+        private readonly IConfiguration configuration;
 
-        public BasicAuthMiddleware(RequestDelegate next)
+        public BasicAuthMiddleware(RequestDelegate next, IConfiguration configuration)

```

Comment:
Can this be made IOptions<theBasicAuthTypeConfig>?

Reply:
Yes, probably a good idea, at the time I was thinking that this would probably be deleted soon, but it might not be